docs/data/product/dea-tasseled-cap-percentiles-landsat/_description.md

```diff
@@ -1,53 +1,96 @@
+- The 10th percentile typically reflects the driest, least vegetated, or least reflective conditions observed during the year.
+- The 50th percentile represents the median state of the landscape, offering a stable reference point for long-term monitoring.
+- The 90th percentile captures peak greenness, wetness, or brightness, useful for identifying maximum vegetation productivity or episodic inundation.
+
+These percentile layers are particularly valuable for land cover classification, wetland mapping, coastal zone monitoring, and ecosystem condition assessment, especially in environments where vegetation, water, and bare ground co-occur or fluctuate seasonally.
+
+The percentile-based approach used in this dataset aligns with best practices in time-series analysis of Earth observation data, as seen in related products such as [DEA Geometric Median and Median Absolute Deviation](https://knowledge.dea.ga.gov.au/data/product/dea-geometric-median-and-median-absolute-deviation-landsat/), [DEA Fractional Cover Percentiles](https://knowledge.dea.ga.gov.au/data/product/dea-fractional-cover-percentiles-landsat/), and [DEA Water Observations from Space (WOFS)](https://knowledge.dea.ga.gov.au/data/product/dea-water-observations-statistics-landsat/). Together, these products form a complementary suite of tools for understanding landscape dynamics across Australia.
 
 ## Lineage
 
-The lineage of this product is inherited from its sole input, the terrain corrected surface reflectance product [Surface Reflectance NBART Collection 3 (Landsat)](https://knowledge.dea.ga.gov.au/data/category/dea-surface-reflectance/).
+The Tasseled Cap Percentiles are derived from Landsat surface reflectance data from the [Surface Reflectance NBART Collection 3 (Landsat)](https://knowledge.dea.ga.gov.au/data/category/dea-surface-reflectance/). This input dataset includes terrain-corrected, analysis-ready data (ARD) from Landsat 5, Landsat 7, Landsat 8, and Landsat 9.
+
+To ensure data quality, all input imagery is masked for cloud and shadow contamination, with a 6-pixel buffer applied around detected cloud and shadow areas. This step reduces the influence of atmospheric interference and improves the reliability of the percentile calculations.
+
+For each calendar year, all valid observations are processed to compute the Tasseled Cap indices using the transformation coefficients described by Crist et al. (1985). These indices are calculated for every 30m x 30m pixel across Australia.
+
+The 10th, 50th, and 90th percentiles are then computed from the full time series of index values for each pixel. These percentiles are statistical summaries, not direct satellite observations. They represent synthetic values that describe the distribution of environmental conditions over the year.
+
+The resulting dataset is a synthetic summary of observed conditions, designed to support environmental monitoring, classification, and change detection across a wide range of Australian ecosystems.
+
```

Comment:
could you keep the lineage section to be concisely on the source datasets and their bands? also move how they're computed to `processing steps` together with the section `Annual summarisation using percentiles`?